### ISSUE TITLE
Using a negative value for dropdownPosition results in a huge blank space at the top of the dropdown on iOS

### ISSUE THREAD

**Issue by kodie** [n4kz/react-native-material-dropdown#76]
As the title states, using a negative value for `dropdownPosition` results in a blank space at the top of the dropdown until you have interacted with it, at which point the blank space disappears. The actual position of the dropdown is perfect, however the blank space is an issue. :(

Issue seems to only happen on iOS as everything looks perfect on Android.

![q3kritfamd](https://user-images.githubusercontent.com/603949/36490311-98a88938-16ed-11e8-9fb4-0b5a8c92d6e6.gif)

react-native v0.52.2
react-native-material-dropdown v0.7.2

Here's the code I'm using:
```javascript
<Dropdown
  label=''
  labelHeight={ 0 }
  baseColor='#fff'
  textColor='#fff'
  itemColor='#fff'
  selectedItemColor='#fff'
  containerStyle={{ backgroundColor: '#932f61', borderColor: '#9d3470', borderWidth: 1, marginVertical: 20, padding: 10 }}
  pickerStyle={{ backgroundColor: '#932f61', borderColor: '#9d3470', borderWidth: 1 }}
  inputContainerStyle={{ borderBottomWidth: 0, marginBottom: -10 }}
  rippleInsets={{ bottom: -10, top: -10, left: 0, right: 0 }}
  dropdownPosition={ -1 }
  data={ this.props.data }
  value={ this.props.value }
  onChangeText={ this.props.onChange }
/>
```

**Comment by kodie** [comment]
Just updated react-native-material-dropdown to v0.9.0 and the issue is still there.

**Comment by kodie** [comment]
Oddly enough, setting `dropdownPosition` to `{ this.props.data.length - 1 }` results in the same display issue.

**Comment by kodie** [comment]
It seems that this only happens if `value` is `null`. Was able to fix it by setting value to a blank string. :)

**Comment by kodie** [comment]
This is probably related to #69

**Comment by n4kz** [comment]
Thanks for issue! Released fix in `0.10.0`. Overscroll is now disabled for positive and negative `dropdownPosition`.
